File: geodata_vis/python/sub.py
import zmq

context = zmq.Context()
socket = context.socket(zmq.SUB)
socket.connect("tcp://127.0.0.1:8888")

# important !!!
socket.setsockopt_string(zmq.SUBSCRIBE, "")

while True:
    string = socket.recv().decode()
    lat, lon = string.split(",")
    print(lat, lon)

File: geodata_vis/example/pub_sub/sub.js
// subber.js
var zmq = require("zeromq"),
  sock = zmq.socket("sub");

sock.connect("tcp://127.0.0.1:5555");
sock.subscribe("kitty cats");
console.log("Subscriber connected to port 5555");

sock.on("message", function(topic, message) {
  console.log(
    "received a message related to:",
    topic,
    "containing message:",
    message
  );
});